Plots/Polygons/NCL_polyg_18.py

```diff
@@ -0,0 +1,134 @@
+"""
+
+###############################################################################
+# Import packages:
+
+import matplotlib.pyplot as plt
+import matplotlib.patches as mpatches
+import cartopy
+import cartopy.crs as ccrs
+from cartopy.mpl.ticker import LatitudeFormatter, LongitudeFormatter
+import numpy as np
+
+from geocat.viz import util as gvutil
+###############################################################################
+# Define helper function to remove ticks/frames from axes
+
+
+def removeTicks(axis):
+    axis.get_xaxis().set_visible(False)
+    axis.get_yaxis().set_visible(False)
+
+
+###############################################################################
+# Plot map, markers, and polygons
+
+# Set size of figure
+fig = plt.figure(figsize=(10, 10))
+
+# Make grid on figure with 2 rows, 1 column
+grid = plt.GridSpec(2, 20, hspace=0.2, wspace=0.2, figure=fig)
+
+# Make subplot for map
+ax = plt.subplot(grid[:-1, 1:], projection=ccrs.PlateCarree())
+
+# Add continents
+continents = cartopy.feature.NaturalEarthFeature(
+        name="coastline",
+        category="physical",
+        scale="50m",
+        edgecolor="None",
+        facecolor="lightgray",
+)
+ax.add_feature(continents)
+
+# Set map extent
+ax.set_global()
+
+# Create arrays with location of each marker
+lon = np.arange(-160, 160, 20)
+lat = np.arange(-80, 80, 10)
+
+# Create array with marker symbols
+# Matplotlib provides a different set of markers than NCL, so plot appearance differs
+marker = ['.', '+', '*', 'o', 'x', 's', '^', 'v', 'D', '>', '<', 'p', 'h', '8', 'X', 'd']
+
+# Draw markers on diagonal line across graph
+for x in range(len(lon)):
+    ax.plot(lon[x], lat[x], marker=marker[x], color='blue', fillstyle='none', markersize=18, zorder=3)
+
+# Draw small red box in upper center
+ax.add_patch(mpatches.Rectangle(xy=[7, 47], width=9, height=7, facecolor='None', edgecolor='red', alpha=1.0, transform=ccrs.PlateCarree(), zorder=5))
+
+# Draw green window in bottom right
+ax.add_patch(mpatches.Rectangle(xy=[110, -45], width=50, height=35, facecolor='lime', alpha=0.3, transform=ccrs.PlateCarree(), zorder=5))
+
+# Use gvutil function to set the ticks on axes
+gvutil.set_axes_limits_and_ticks(ax, xlim=None, ylim=None, xticks=np.arange(-180, 210, 30), yticks=np.arange(-90, 120, 30), xticklabels=None, yticklabels=None)
+
+# Use gvutil function to give ticks W/N/E/S labels
+gvutil.add_lat_lon_ticklabels(ax, zero_direction_label=True, dateline_direction_label=True)
+
+# Took out degree symbols in latitude/longitude
+ax.yaxis.set_major_formatter(LatitudeFormatter(degree_symbol= ''))
+ax.xaxis.set_major_formatter(LongitudeFormatter(degree_symbol=''))
+
+# Use gvutil function to set title of plot
+# Set title font to bold using the r"$\bf{_____}$" formatting characters
+gvutil.set_titles_and_labels(ax, maintitle=r"$\bf{Big}$"+ " " + r"$\bf{centered}$" + " " +r"$\bf{title}$", maintitlefontsize=25)
```

Comment:
Is there a specific reason to handle the maintitle string separately for each word?

Reply:
Yes, for some reason when using the font-bolding escape characters the spaces don't show up when they're inside the curly braces. I can comment that in.